src/models/modelUtils.js

```diff
@@ -18,10 +18,17 @@ const lengthBetweenValidator = (val, min, max) => {
     return true;
 };
 
+const validImageURL = (val) => {
```

Comment:
I'm pretty certain it is like this. Example: https://res.cloudinary.com/demo/image/upload/q_60/sample.jpg

Reply:
I can confirm that (from their API docs: https://cloudinary.com/documentation/image_upload_api_reference#sample_response).

However, I think it's wiser to just validate that it is a URL (no extension required) in case we ever change the Image Host to something else that doesn't have it. We would probably forget this and it would cause errors

Reply:
Addressed requested changes